alg/teca_latitude_damper.cxx

```diff
@@ -152,6 +213,16 @@ std::vector<teca_metadata> teca_latitude_damper::get_upstream_request(
 
     arrays.insert(damped_vars.begin(), damped_vars.end());
 
+    // Cleaning off the postfix for arrays passed in the pipeline. 
+    // For ex a down stream could request "foo_damped" then we'd
+    // need to request "foo". also remove "foo_damped" from the
+    // request.
+    std::string var_post_fix = this->variable_post_fix;
```

Comment:
Hi Abdel, I'm seeing that we are making a copy of this string in a couple of the changes. it would be better to take a const reference to it, or just use the member variable directly. doing one of those would avoid copying the string.